### DISCUSSION TITLE
Icon for Calc Groups doesn't show in Auto-Complete dialog

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1324] [Ideas]
### Description

When writing a measure, or any expression, I need to be able to distinguish between normal tables and calc groups. Consider the following:
![image](https://github.com/TabularEditor/TabularEditor3/assets/45298358/c2d15836-a9e1-4e1d-8026-020d458ebe29)

I need to trap the visible values in the Relative Period slicer. Because the tables are not clearly labled or easily distinguished, I am not sure if I need the Relative Period table or the Relative Period Selection table. Only the presence of the Ordinal field gives a clue here.

Ideally, the icon would show the calc group icon in the Autocomplete area.
![image](https://github.com/TabularEditor/TabularEditor3/assets/45298358/c139e4fb-7545-4f70-a4e8-022634e8e653)

### Tabular Editor 3 Version

3.16

### Windows Version

11

**Comment by mlonsk** [comment]
Hi Ed
I definitely see the need for this change. 
I will, however, move this to our ideas forum as it is not strictly speaking a bug but rather an enhancement request.
